src/foam/dao/SinkJava.js

```diff
@@ -111,21 +111,23 @@ foam.CLASS({
   methods: [
     {
       name: 'put',
-      javaCode: 'if ( getCount() >= getLimit() ) {\n'
-              + '  if ( sub != null ) sub.detach();\n'
-              + '} else {\n'
-              + '  setCount(getCount() + 1);\n'
-              + '  getDelegate().put(obj, sub);\n'
-              + '}\n'
+      javaCode: 
+`if ( getCount() >= getLimit() ) {
+  if ( sub != null ) sub.detach();
+} else {
+  setCount(getCount() + 1);
+  getDelegate().put(obj, sub);
+}`
     },
     {
       name: 'remove',
-      javaCode: 'if ( getCount() >= getLimit() ) {\n'
-              + '  if ( sub != null ) sub.detach();\n'
-              + '} else {'
-              + '  setCount(getCount() + 1);\n'
-              + '  getDelegate().put(obj, sub);\n'
-              + '}\n'
+      javaCode: 
```

Comment:
FYI: Best not to mix formatting and code changes in same PR. Then for the formatting changes are easier to review.